src/eventkit.cs

```diff
@@ -200,17 +200,21 @@ interface EKAlarm : NSCopying {
 		EKAlarmProximity Proximity { get; set;  }
 
 #if MONOMAC
+		[NoMacCatalyst]
```

Comment:
There are numerous `[NoMacCatalyst]` attributes that are inside `#if MONOMAC`, they're all redundant.